### DISCUSSION TITLE
what JavaScript file and what part of it, to display the source code that doxygen produce ?

### DISCUSSION THREAD

**Discussion by paule32** [doxygen/doxygen#11174] [Q&A]
Hello,
which JavaScript file is used to display the source code (with line numbers) ?
How can I implement it in own sources ?

**Comment by albert-github** [comment]
Do you mean the part when `SOURCE_BROWSER = YES` or `\snippet` or `\include`?
These don't use javascript to display the, highlighted, source code and the linenumbers but the c++ code to craete the different files: html / latex / xml / ....

**Comment by paule32** [comment]
Thank you for reply.
